src/modules/cli/hooks/useCLI.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  BitCoinIcon,
+  CoinsIcon,
+  MoreLessIcon,
+  RecoveryIcon,
+} from '@/components';
+
+import { TabState, useAPIToken } from './APIToken';
+
+const useCLI = () => {
+  const { dialog, steps, tabs, create, remove, list, hasToken } = useAPIToken();
+
+  const settings = [
+    {
+      label: 'API Tokens',
+      icon: CoinsIcon,
+      disabled: false,
+      onClick: () => {
+        if (hasToken) {
```

Comment:
change to more simple:

`hasToken ? tabs.set(TabState.LIST) : tabs.set(TabState.CREATE)`